### ISSUE TITLE
The Launcher doesn't understand nested directories in the scene folder

### ISSUE THREAD

**Issue by alexanderbock** [OpenSpace/OpenSpace#126]

**Comment by alexanderbock** [comment]
It actually does understand nested directories already, but it's rather a problem of multiple file requests going to the same destination directory:
In `scene/newhorizons/pluto/pluto/pluto.data`:
```lua
        { Identifier = "newhorizons_plutoencounter_pluto_textures", Destination = "textures", Version = 3 },
        { Identifier = "pluto_textures", Destination = "textures", Version = 2 },
```

**Comment by alexanderbock** [comment]
Closed in 21e959988828aea6f31ae86605f97b09fbc5a5fc

Reason: SyncWidget is removing duplicates and was ignoring the base directory, thus removing files that would have shown up in two different locations
